.pre-commit-config.yaml

```diff
@@ -9,9 +9,11 @@ repos:
         stages: [commit-msg]
```

Comment:
the `stages` key is now deprecated, and defaults to our config, so you can remove all of these `stages` keys.

Reply:
(except for the commit-msg one, which is not default)

Reply:
okay i am now less sure of this change, so please test it, but i think it would help clean up this file a bit.

Reply:
ok i confirmed the behavior -- it is save to remove the stages keys.